src/govuk-prototype-kit.config.json

```diff
@@ -1,4 +1,10 @@
 {
+  "pluginDependencies": [
+    {
+      "packageName": "govuk-frontend",
+      "minVersion": "4.7.0"
+    }
+  ],
```

Comment:
when you install hmrc-frontend through the prototype kit UI this will prompt the designer that govuk-frontend is also needed (if it isn't already installed) and it can then install it for them. Otherwise if govuk-frontend wasn't installed someone could install hmrc-frontend without realising they also needed govuk-frontend and not get stuff autowired together with the prototype kit conventions. It's not a likely scenario at the moment, but here's an example of what the prompt looks like when installing via the web ui of the prototype kit:
![CleanShot 2023-08-01 at 11 36 02@2x](https://github.com/hmrc/hmrc-frontend/assets/100650/77f7c88a-5a90-4bb6-bb71-cee274d38531)